docs/release-notes.rst

```diff
@@ -93,6 +93,11 @@ Improvements
   serialized non-deterministically, which could lead to downstream values being
   spuriously recomputed. (Unfortunately this fix doesn't help with other objects that
   happen to contain sets.)
+- Clarified an `error message <https://github.com/square/bionic/issues/331>`__ that
+  occurs when users pass a single argument to @bn.outputs instead of using @bn.output.
```

Comment:
nit: we generally write decorators, etc., in monospace, like this:

    ``@bn.outputs``